integration-tests/src/tests/client/sync_state_nodes.rs

```diff
@@ -744,10 +745,17 @@ fn test_dump_epoch_missing_chunk_in_last_block() {
                         .chain
                         .get_chunk_extra(blocks[height as usize].hash(), &ShardUId::single_shard())
                         .unwrap();
+                    let expected_height = if ProtocolFeature::BandwidthScheduler
+                        .enabled(genesis.config.protocol_version)
+                    {
+                        height
+                    } else {
+                        last_chunk_height
+                    };
```

Comment:
Do I want to know?

Reply:
I wrote about it in the commit description